Server/src/main/java/org/gluu/oxauth/authorize/ws/rs/AuthorizeRestWebServiceImpl.java

```diff
@@ -142,13 +145,13 @@ public Response requestAuthorizationGet(
             String scope, String responseType, String clientId, String redirectUri, String state, String responseMode,
             String nonce, String display, String prompt, Integer maxAge, String uiLocales, String idTokenHint,
             String loginHint, String acrValues, String amrValues, String request, String requestUri,
-            String requestSessionId, String sessionId, String originHeaders,
-            String codeChallenge, String codeChallengeMethod, String customResponseHeaders, String claims, String authReqId,
+            String requestSessionId, String sessionId, String originHeaders, String codeChallenge,
+            String codeChallengeMethod, String customResponseHeaders, String claims, String authReqId, String userCode,
             HttpServletRequest httpRequest, HttpServletResponse httpResponse, SecurityContext securityContext) {
         return requestAuthorization(scope, responseType, clientId, redirectUri, state, responseMode, nonce, display,
                 prompt, maxAge, uiLocales, idTokenHint, loginHint, acrValues, amrValues, request, requestUri,
                 requestSessionId, sessionId, HttpMethod.GET, originHeaders, codeChallenge, codeChallengeMethod,
-                customResponseHeaders, claims, authReqId, httpRequest, httpResponse, securityContext);
+                customResponseHeaders, claims, authReqId, userCode, httpRequest, httpResponse, securityContext);
```

Comment:
Because authorization is done also using this and authorize page, during the whole process the identifier for device authz is userCode, it could be expired and also it allows get information from cache when authz is granted or denied.